src/Umbraco.Web.UI.Client/src/common/directives/components/umbminisearch.component.js

```diff
@@ -21,6 +22,7 @@
 
         vm.onKeyDown = onKeyDown;
         vm.onChange = onChange;
+        vm.randomUniquId = randomUniquId();
```

Comment:
@BatJan why not using `String.CreateGuid()` here to keep it consistent?
https://github.com/umbraco/Umbraco-CMS/blob/d428a4543f33bb7094cf7db5f6b6fdc2d1de3063/src/Umbraco.Web.UI.Client/src/views/propertyeditors/radiobuttons/radiobuttons.controller.js#L10